Neos.Flow/Classes/Mvc/ActionResponse.php

```diff
@@ -0,0 +1,104 @@
+<?php
+namespace Neos\Flow\Mvc;
+
+use Neos\Flow\Annotations as Flow;
+use Psr\Http\Message\UriInterface;
+
+/**
+ * The new minimal MVC response object.
+ * For anything more use a custom HTTP component and set a component parameter.
+ *
+ * @Flow\Proxy(false)
+ * @api
+ */
+final class ActionResponse extends \Neos\Flow\Http\Response
+{
+    use ResponseDeprecationTrait;
+
+    /**
+     * @var array
+     */
+    private $componentParameters = [];
+
+    /**
+     * @var UriInterface
+     */
+    private $redirectUri;
+
+    /**
+     * @var string
+     */
+    private $contentType = '';
+
+    /**
+     * @param string $content
+     * @return void
+     * @api
+     */
+    public function setContent($content): void
```

Comment:
```suggestion
    public function setContent(string $content): void
```

Reply:
I think we decided to go more for streaminterface here, so not commiting this.